CMakeLists.txt

```diff
@@ -611,6 +611,7 @@ if(ENABLE_EXAMPLE)
 endif()
 
 if(ENABLE_DOCS)
+  set(DOC_LANG en CACHE STRING "Document language option")
```

Comment:
Either move this set outside of ENALBE_DOCS here or add a conditional below for `cmake_print_variables` or it will be printing empty except for a docs build.

Reply:
decided to check DOC_LANG before printing.